.github/workflows/cd-sql-engine-populate-persistent-source-schema.yaml

```diff
@@ -0,0 +1,106 @@
+# See [Persistent Source Schema](/GLOSSARY.md#persistent-source-schema)
+# Populating the source schema via this workflow ensures that it's done with the same settings as the tests.
+
+name: Reload Test Data in SQL Engines
+
+# We don't want multiple workflows trying to create the same table.
+concurrency:
+  group: POPULATE_PERSISTENT_SOURCE_SCHEMA
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
Please add `workflow_dispatch` so we don't have to label random PRs if something gets borked off main.

Reply:
Added, but iterating on this PR has been a huge pain as it can't be tested locally.